lib/trento/application/usecases/sap_systems/health_summary_service.ex

```diff
@@ -48,10 +48,26 @@ defmodule Trento.SapSystems.HealthSummaryService do
       sapsystem_health: health,
       database_health: compute_database_health(database_instances),
       clusters_health: compute_clusters_health(all_instances),
-      hosts_health: compute_hosts_health(all_instances)
+      hosts_health: compute_hosts_health(all_instances),
+      hana_cluster_id: extract_hana_cluster_id(database_instances),
+      database_id: extract_database_id(database_instances)
     })
   end
 
+  @spec extract_database_id([DatabaseInstanceReadModel.t()]) :: String.t()
```

Comment:
I would put the `extract_*` functions after the `compute_*` ones. To have them sorted as the appear in the code

Reply:
+ 1

Reply:
done